tsconfig.base.json

```
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
```

also your call on this but I've been using this as a handy reference lately: https://www.totaltypescript.com/tsconfig-cheat-sheet#building-for-a-library-in-a-monorepo

and it recommends adding the following options in monorepos:

```suggestion
    "allowJs": true,
    "composite": true,
    "sourceMap": true,
    "declarationMap": true
```